mx.sulong/mx_sulong.py

```diff
@@ -573,10 +573,9 @@ def runLLVMTestCases(args=None):
 
 def runTruffleTestCases(args=None):
     """runs the Sulong test suite"""
-    ensureLLVMBinariesExist()
-    ensureDragonEggExists()
+    testsuites.compileSuite(['sulong'])
     vmArgs, _ = truffle_extract_VM_args(args)
-    return unittest(getCommonUnitTestOptions() + ['-Dsulong.ExecutionCount=1000'] + vmArgs + ["com.oracle.truffle.llvm.test.SulongTestSuite"])
+    return unittest(vmArgs + ["com.oracle.truffle.llvm.test.alpha.SulongSuite"])
```

Comment:
OK, I agree.
But rather than running tests 1000x I suggest:
graal.TruffleCompileImmediately=true in combination with TruffleBackgroundCompilation=false

This actually tests more because you cannot run a method without compilation.
I need to find out how much time this takes.